tools/cpp/unix_cc_toolchain_config.bzl

```diff
@@ -1275,6 +1293,17 @@ def _impl(ctx):
         ],
     )
 
+    macos_minimum_os_feature = feature(
+        name = "macos_minimum_os",
+        enabled = True,
+        flag_sets = [
+            flag_set(
+                actions = all_compile_actions,
+                flag_groups = [flag_group(flags = ["-mmacos-version-min={}".format(_target_os_version(ctx))])],
```

Comment:
this is required because when you have Xcode + the CLT installed, it seems that the minimum os version behavior changes somehow, without explicitly passing this I saw it defaulting to the oldest version. This also means `--macos_minimum_os` works for C* binaries

Reply:
Definitely make this a comment.